builder/builder.go

```diff
@@ -53,23 +66,40 @@ func (b *Builder) Build() error {
 	layerIDs := []string{}
 	layers := []mml.Layer{}
 
-	if b.mml != "" {
-		r, err := os.Open(b.mml)
+	if b.mmlFile != "" {
+		r, err := os.Open(b.mmlFile)
 		if err != nil {
 			return err
 		}
 		defer r.Close()
-		mml, err := mml.Parse(r)
+		b.mmlData, err = mml.Parse(r)
```

Comment:
There is no need to set mmlData. Just keep the old code: if mmlFile is set, load into mml. If mmlData is set, assign mml to this value.
Then apply baseDir to the Stylesheet filenames. And the patch should be complete, or am I missing something?